app/models/item.rb

```diff
@@ -90,7 +90,7 @@ def decrypt_attr(attr, team_password)
   end
 
   def encrypt_file(team_password)
-    return if cleartext_file.blank?
+    return if cleartext_file.force_encoding('ISO-8859-1').blank?
```

Comment:
what the encoding for ?

Reply:
actually just updated that. Was a bug because it tried to use a match on an invalid utf-8 string and thus encountered an ArgumentError.
Replaced blank? with empty? now and it works...